tez-dag/src/main/java/org/apache/tez/dag/app/DAGAppMaster.java

```diff
@@ -1743,6 +1747,13 @@ public void setQueueName(String queueName) {
     }
   }
 
+  private String getShutdownTimeString() {
+    if (shutdownHandler != null && shutdownHandler.shutdownTime != null) {
+      return ". The shutdown hook started at " + shutdownHandler.shutdownTime;
```

Comment:
It's strange that this separate string generation method takes care of the closing point of the previous sentence, I believe that should go back to  getApplicationACLs

Reply:
I moved the "." back to the {{getApplicationACLs}}